apps/Box86/uninstall

```diff
@@ -12,6 +12,14 @@ cd ~/box86/build
 sudo make uninstall
 cd $HOME
 
+CODENAME=$(lsb_release -cs)
+if [ "${CODENAME}" = "focal" || "${CODENAME}" = "buster" ];then
+  "${DIRECTORY}/purge-installed" "libc6:armhf  libx11-6:armhf  libgdk-pixbuf2.0-0:armhf libgtk2.0-0:armhf libstdc++6:armhf libsdl2-2.0-0:armhf mesa-va-drivers:armhf libsdl1.2-dev:armhf libsdl-mixer1.2:armhf libpng16-16:armhf libcal3d12v5:armhf libsdl2-net-2.0-0:armhf libopenal1:armhf libsdl2-image-2.0-0:armhf libvorbis-dev:armhf libcurl4:armhf osspd:armhf pulseaudio:armhf libjpeg62:armhf libudev1:armhf libgl1-mesa-dev:armhf libsnappy1v5:armhf libx11-dev:armhf libsmpeg0:armhf libboost-filesystem1.67.0:armhf libboost-program-options1.67.0:armhf libavcodec58:armhf libavformat58:armhf libswscale5:armhf libmyguiengine3debian1v5:armhf libboost-iostreams1.67.0:armhf  libsdl2-mixer-2.0-0:armhf" || exit 1
```

Comment:
no, thast not how `purge-installed` works (I made the same mistake once), use this instead:
```bash
if [ ! -z "$(file "$(readlink -f "/sbin/init")" | grep 64)" ]; then
    "${DIRECTORY}/purge-installed" "$(dirname "$0")" || exit 1
elif [ ! -z "$(file "$(readlink -f "/sbin/init")" | grep 32)" ]; then
    sleep 0.00001
else
    error "Can't detect OS architecture! something is very wrong!"
fi
```